packages/core/src/external/carequality/ihe-gateway-v2/outbound/xcpd/create/iti55-envelope.ts

```diff
@@ -221,11 +226,10 @@ function createSoapBody({
   const providerId = bodyData.principalCareProviderIds[0];
   const homeCommunityId = getHomeCommunityId(gateway, bodyData.samlAttributes);
   const patientGender = bodyData.patientResource.gender === "female" ? "F" : "M";
-  const patientBirthtime = bodyData.patientResource.birthDate.replace(DATE_DASHES_REGEX, "");
-  const patientFamilyName = bodyData.patientResource.name?.[0]?.family;
-  const patientGivenName = bodyData.patientResource.name?.[0]?.given?.[0];
-  const patientAddress = bodyData.patientResource.address?.[0];
-  const patientTelecom = bodyData.patientResource.telecom?.[0]?.value ?? undefined;
+  const patientBirthtime = bodyData.patientResource.birthDate?.replace(DATE_DASHES_REGEX, "");
+  const patientName = bodyData.patientResource.name;
+  const patientAddress = bodyData.patientResource.address;
+  const patientTelecom = bodyData.patientResource.telecom;
```

Comment:
we havent been using all the patients addresses or names, and havent been using their telecoms!